code/modules/power/tesla/energy_ball.dm

```diff
@@ -1,6 +1,15 @@
 #define TESLA_DEFAULT_POWER 1738260
 #define TESLA_MINI_POWER 869130
-
+//Zap constants, speeds up targeting
+#define COIL (ROD + 1)
+#define ROD (RIDE + 1)
+#define RIDE (LIVING + 1)
+#define LIVING (MACHINERY + 1)
+#define MACHINERY (BLOB + 1)
+#define BLOB (STRUCTURE + 1)
+#define STRUCTURE (1)
```

Comment:
Given these appear in the SM and the tesla, should these be globally scoped as `ZAPCONSTANT_COIL` or whatever? Just incase someone updates them in one plate not another

Reply:
The Tesla and SM shocking are actually different, if you look closely at the defines.